import_map.json

```diff
@@ -1,5 +1,6 @@
 {
   "imports": {
-    "lume/": "https://deno.land/x/lume@v1.11.2/"
+    "lume/": "https://deno.land/x/lume@v1.11.2/",
+    "dbin/": "https://deno.land/x/dbin@v0.1.1/"
```

Comment:
Don't include plugin dependencies in the import_map file, because it's harder to install and upgrade later. Here there should be only the `lume/` import.